schemas/eq_submission_schema.json

```diff
@@ -295,54 +283,264 @@
+                          "pattern": "^(.*)$"
+                        },
+                        "list_item_id": {
+                          "$id": "#/properties/data/properties/answers/items/properties/list_item_id",
+                          "type": "string",
+                          "title": "List Item ID",
+                          "description": "Related item will have the same ID e.g. first name and last name for the same person",
+                          "default": "",
+                          "examples": [
+                            "zGBdpb"
+                          ],
+                          "pattern": "^(.*)$"
+                        }
+                      }
+
+                  },
+                  {
+                      "$id": "#/properties/data/properties/answers/items/anyOf/1",
+                      "type": "object",
+                      "title": "Address Answer Type",
+                      "examples": [
+                          {
+                              "answer_id": "address-answer",
+                              "value": {
+                                  "line1": "Address Line 1",
+                                  "town": "Town",
+                                  "postcode": "WA10 1AB",
+                                  "uprn": "1"
+                              },
+                              "list_item_id": "abcde"
+                          }
+                      ],
+                      "required": [
+                          "answer_id",
+                          "value"
+                      ],
+                      "properties": {
+                          "answer_id": {
+                              "$id": "#/properties/data/properties/answers/items/anyOf/1/properties/answer_id",
+                              "type": "string",
+                              "title": "Answer ID",
+                              "default": "",
+                              "examples": [
+                                  "address-answer"
+                              ],
+                              "pattern": "^(.*)$"
+                          },
+                          "value": {
+                              "$id": "#/properties/data/properties/answers/items/anyOf/1/properties/value",
+                              "type": "object",
+                              "title": "Address Answer Value",
+                              "default": {},
+                              "examples": [
+                                  {
+                                      "line1": "Address Line 1",
+                                      "town": "Town",
+                                      "postcode": "NP10 8XG",
+                                      "uprn": "12345678912"
+                                  }
+                              ],
+                              "required": [
+                                  "line1",
+                                  "town",
```

Comment:
Only line1 is mandatory for our current runner address answer type implementation.